ui/effects/use-on-resize.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+import debounce from 'util/debounce';
+
+export function useOnResize(cb) {
+  const isWindowClient = typeof window === 'object';
```

Comment:
What scenario is this line for?
`const isWindowClient = typeof window === 'object';`

Reply:
I saw this in [ui/effects/use-screensize.js](https://github.com/OdyseeTeam/odysee-frontend/pull/774/files/9005787a4d8e09c0438f4f44b08dfd92c8ad9855#diff-e323bca944c6ea37655fa4af12284705e90487aa69dffc2efa8cb3042e97ab96) and assumed this was either for SSR or some electron scenario that might need to be catered to.